projenrc/support.ts

```diff
@@ -2,12 +2,13 @@ import { JsonFile, Project } from 'projen';
 import type { ReleasesDocument } from '../src/support';
 
 export const SUPPORT_POLICY: ReleasesDocument = {
-  current: '5.3',
+  current: '5.4',
   maintenance: {
     // version: End-of-support date
     '5.0': new Date('2024-01-31'),
     '5.1': new Date('2024-02-28'),
     '5.2': new Date('2024-06-30'),
+    '5.3': new Date('2024-10-07'),
```

Comment:
This is fine, but given that it might take a day we tended to give a few days extra. Looks like so far we have rounded up to the end of the month. So maybe 15.?
```suggestion
    '5.3': new Date('2024-10-15'),
```